templates/prescription/general_list.html

```diff
@@ -83,6 +83,33 @@
     }
 }
 
+function deleteDrug(d) {
+    let isPromise = top.jsFetchGlobals('e_rx');
+    isPromise.then(
+        msg => {
+            alert('msg.e_rx.deleteWarn');
+            if (choice == true) {
+                $.ajax({
+                    url: "./library/deleteDrug.php",
+                    type: 'POST',
+                    data: {
+                        drugId: d,
+                        csrf_token_form: {$CSRF_TOKEN_FORM}
```

Comment:
escape this:
```
csrf_token_form: {$CSRF_TOKEN_FORM|js_escape}
```

Reply:
yep, sorry Sherwin. This is way to go but hey, we learned something....

Reply:
updated.